File: C/beginner/1078.c
// https://judge.beecrowd.com/pt/problems/view/1078
// Tabuada

// Libraries
#include <stdio.h>

// Main Function
int main(){
    // Starting Code
    int user_input;

    // Scanning input
    scanf("%d", &user_input);

    // Iterating
    for (int i = 1; i <= 10; i++){
        printf("%d x %d = %d\n", user_input, i, user_input * i);
    }

    // Ending Code
    return 0;
}

// By Fernando Serra
// https://github.com/fernandosserra

File: Haskell/beginner/1078.hs
-- https://judge.beecrowd.com/pt/problems/view/1078
-- Tabuada

-- Main Function
main :: IO ()
main = do
    -- Starting code and scanning input
    user_input <- readLn :: IO Int
    -- Iterating
    mapM_ (printMultiplication user_input) [1..10]

-- Function to print multiplication
printMultiplication :: Int -> Int -> IO ()
printMultiplication user_input i = 
    putStrLn $ show i ++ " x " ++ show user_input ++ " = " ++ show (user_input * i)

-- By Fernando Serra
-- https://github.com/fernandosserra